### ISSUE TITLE
`GraphvizSetAttr` doesn't work on 2nd application

### ISSUE THREAD

**Issue by james-d-mitchell** [digraphs/graphviz#24]
~~~gap
gap> gv := GraphvizGraph("xxx");
<graphviz graph xxx with 0 nodes and 0 edges>
gap> GraphvizAddNode(gv, 1);
<graphviz node 1>
gap> GraphvizAddNode(gv, 2);
<graphviz node 2>
gap> GraphvizAddNode(gv, 3);
<graphviz node 3>
gap> GraphvizSetNodeLabels(gv, ["i", "ii", "iii"]);
<graphviz graph xxx with 3 nodes and 0 edges>
gap> Print(String(gv));
//dot
graph xxx {
	1 [label=i]
	2 [label=ii]
	3 [label=iii]
}
gap> GraphvizSetNodeLabels(gv, ["a", "b", "c"]);
<graphviz graph xxx with 3 nodes and 0 edges>
gap> Print(String(gv));
//dot
graph xxx {
	1 [label=i]
	2 [label=ii]
	3 [label=iii]
}
~~~

Should rather be "a", "b", and "c" the second time around.

**Comment by james-d-mitchell** [comment]
Here's another example:

~~~
gap> gv := GraphvizGraph("xxx");
<graphviz graph xxx with 0 nodes and 0 edges>
gap> GraphvizAddNode(gv, 1);
<graphviz node 1>
gap> GraphvizAddNode(gv, 2);
<graphviz node 2>
gap> GraphvizAddNode(gv, 3);
<graphviz node 3>
gap> GraphvizSetNodeColors(gv, ["i", "ii", "iii"]);
Error, invalid color "i" (list (string)), valid colors are RGB values or names\
 from the GraphViz 2.44.1 X11 Color Scheme http://graphviz.org/doc/info/colors\
.html
gap> GraphvizSetNodeColors(gv, ["red", "green", "blue"]);
<graphviz graph xxx with 3 nodes and 0 edges>
gap> Print(String(gv));
//dot
graph xxx {
	1 [color=red, style=filled]
	2 [color=green, style=filled]
	3 [color=blue, style=filled]
}
gap> GraphvizSetNodeColors(gv, ["red", "#00FF00", "blue"]);
<graphviz graph xxx with 3 nodes and 0 edges>
gap> Print(String(gv));
//dot
graph xxx {
	1 [color=red, style=filled]
	2 [color=green, style=filled]
	3 [color=blue, style=filled]
}
gap> GraphvizSetNodeColors(gv, ["#FF0000", "#00FF00", "#0000FF"]);
<graphviz graph xxx with 3 nodes and 0 edges>
gap> Print(String(gv));
//dot
graph xxx {
	1 [color=red, style=filled]
	2 [color=green, style=filled]
	3 [color=blue, style=filled]
}
~~~

**Comment by james-d-mitchell** [comment]
I think this is a problem that I introduced, will fix.

**Comment by james-d-mitchell** [comment]
Resolved it in my PR #21
